src/matter/cluster/BasicCluster.ts

```diff
@@ -4,17 +4,43 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { StringT, UnsignedIntT } from "../../codec/TlvObjectCodec";
+import { StringT, UnsignedIntT, BooleanT, Field, ObjectT } from "../../codec/TlvObjectCodec";
 import { AttributeSpec, ClusterSpec } from "./ClusterSpec";
 
+const CapabilityMinimaStructT = ObjectT({
+  caseSessionsPerFabric: Field(0, UnsignedIntT),
+  subscriptionsPerFabric: Field(1, UnsignedIntT),
+});
+
+/**
+ * This cluster provides attributes and events for determining basic information about Nodes, which supports both
+ * Commissioning and operational determination of Node characteristics, such as Vendor ID, Product ID and serial number,
+ * which apply to the whole Node. Also allows setting user device information such as location.
+ */
 export const BasicClusterSpec = ClusterSpec(
     0x28,
     "Basic",
     {
```

Comment:
Per the specs, some attributes are read-only, some are writable with various ACL level.
Maybe you can add in a comment what is what: this will help when this will be refactored to add read / write control and ACLs

Reply:
WIll do

Reply:
Please check below ... that way or do you like it differently better?